src/web_interface/templates/database/database_binary_search.html

```diff
@@ -98,6 +93,45 @@ <h5 style="color: red;">{{ error }}</h5>
             </div>
         </form>
     </div>
+    <div class="col-md-offset-2 col-md-8 col-xs-12">
+		<h4>Example queries:</h4>
+		<p>
+            HEX-Pattern:<br />
+	        <pre><code>rule a_hex_string_rule
```

Comment:
seems like `<pre>` in `<p>` doesn't work and the paragraph gets closed automatically
therefore this should be equivalent:
```html
<p>HEX-Pattern:</p>
<pre><code>
(...)
```